docs/en/02_defining-jobs.md

```diff
@@ -1,72 +1,77 @@
-# Defining Jobs
+---
+title: Defining Jobs
+summary: Creating custom queued jobs, covering the API, constructors, job state management, multiple steps, and advanced setup patterns.
+icon: cogs
+---
+
+# Defining jobs
 
 The best way to learn about defining your own jobs is by checking the examples
 
-* `PublishItemsJob` - A job used to publish all the children of a particular node. To create this job, run the PublishItemsTask passing in the parent as a request var (eg ?parent=1)
-* `GenerateGoogleSitemapJob` - A job used to create a google sitemap. If the googlesitemaps module is installed it will include priority settings as defined there, otherwise just produces a generic structure. To create an initial instance of this job, call dev/tasks/CreateDummyJob?name=GenerateGoogleSitemapJob. This will create the initial job and queue it; once the job has been run once, it will automatically schedule itself to be run again 24 hours later. 
-* `CreateDummyJob` - A very simple skeleton job. 
+- [`PublishItemsJob`](api:Symbiote\QueuedJobs\Jobs\PublishItemsJob) - A job used to publish all the children of a particular node. To create this job, run the PublishItemsTask passing in the parent as a request var (eg ?parent=1)
+- [`GenerateGoogleSitemapJob`](api:Symbiote\QueuedJobs\Jobs\GenerateGoogleSitemapJob) - A job used to create a google sitemap. If the googlesitemaps module is installed it will include priority settings as defined there, otherwise just produces a generic structure. To create an initial instance of this job, call `dev/tasks/CreateDummyJob?name=GenerateGoogleSitemapJob`. This will create the initial job and queue it; once the job has been run once, it will automatically schedule itself to be run again 24 hours later.
 
-## API Overview
+## API overview
 
-The module comes with an `AbstractQueuedJob` class that defines many of the boilerplate functionality for the
-job to execute within the framework. An example job can be found in queuedjobs/code/jobs/PublishItemsJob.php.
+The module comes with an [`AbstractQueuedJob`](api:Symbiote\QueuedJobs\Services\AbstractQueuedJob) class that defines many of the boilerplate functionality for the
+job to execute within the framework. An example job can be found in [`PublishItemsJob.php`](https://github.com/silverstripe/silverstripe-queuedjobs/blob/6/src/Jobs/PublishItemsJob.php).
```

Comment:
Probably better to use an API link and say "in the source code for ``[`PublishItemsJob`](api:etc\PublishItemsJob)``"

Reply:
Done